src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -0,0 +1,262 @@
+        :id: 6ec16a9a-ca03-11ec-9675-845cf3eff344
+        :steps:
+          1. Non existing netgroup returned with proxy provider
+         when proxy lib name is file
+        :expectedresults:
+          2. Should not succeed
+        """
+        tools = sssdTools(multihost.client[0])
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'use_fully_qualified_names': 'False',
+                         'id_provider': 'proxy',
+                         'auth_provider': 'proxy',
+                         'cache_credentials': 'true',
+                         'proxy_lib_name': 'files',
+                         'proxy_pam_target': 'sssdproxyldap'}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "getent netgroup testsumgroup")
+        execute_cmd(multihost, "echo 'QAeng    "
+                               "(host1.example.com, ami1, example.com)'"
+                               " > /etc/netgroup")
+        tools.clear_sssd_cache()
+        execute_cmd(multihost, "getent netgroup QAeng")
+        execute_cmd(multihost, 'echo "" > /etc/netgroup')
+
+    def test_bz647816(self, multihost, backupsssdconf):
+        """
+        :title: More than 10 auth attempt times out bz647816
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=647816
+        :id: 201a44c0-b252-11ec-94b7-845cf3eff344
+        :steps:
+            1. Auth a user more than 12 times
+            2. Search for Error
+        :expectedresults:
+            1. Should succeed
+            2. Should not succeed
+        """
+        # more than 10 auth attempt times out bz647816
+        tools = sssdTools(multihost.client[0])
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'proxy_lib_name': 'ldap',
+                         'debug_level': '0xFFF0'}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        client_e = multihost.client[0].ip
+        for _ in range(12):
+            ssh1 = SSHClient(client_e, username="foo1@example1",
+                             password="Secret123")
+            ssh1.close()
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "grep 'All available child slots are full, "
+                                   "queuing request' /var/log/sssd/*")
+
+    def test_0002_bz1209483(self, multihost, backupsssdconf):
+        """
+        :title: sssd does not work as expected when id provider
+         equal to proxy and auth provider equal to ldap bz1209483
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1209483
+        :id: 2fa4834c-b252-11ec-919d-845cf3eff344
+        :steps:
+          1. Adding the user in local
+          2. sssd does not work as expected when id provider equal to proxy
```

Comment:
Maybe add 2 steps 
```
2. set id provider equal to proxy and auth provider equal to ldap
3. lookup local user
```

Reply:
is it needed that there should be a user in ldap with the same name? then it should be checked that a user with the same name exists in ldap.

Reply:
Done And foo2 is added in the fixture "misc_test"

Reply:
Yes the user exists from `create_posix_usersgroups` fixture, but the test step should include to check that a user with same name already exists in ldap to make it clear that it is an important part of the test.